### PR TITLE
Refactoring & fixes

### PR DESCRIPTION
**Pull request by navstyachka**
This pull makes the Composer HTML viewer ready to use.
The changes include minor refactoring and fixes:
Added file copying into build folder;
Fixed paths for static assets;
Added 'Powered by Juno';
Commented search bar until developed;

### CODE REVIEW

File: html-report/src/components/SuitesList.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default class SuitesList extends Component {
                     return (<li key={ i } className="list__item no-hover">
                         <div className="text-title margin-bottom-10 label">{ device.id }</div>
                         <div className="margin-bottom-10">
-                          <a href={ `../logs/${device.logcat_path}` }>{ device.logcat_path }</a>
+                          <a href={ `${device.logcat_path}` }>{ device.logcat_path }</a>
```
Comment (navstyachka) [inline html-report/src/components/SuitesList.js]: @artem-zinnatullin Please pay attention that I've changed the path

File: html-report/index.html [hunk 2]
```diff
@@ -14,5 +14,6 @@
     <div id="root"></div>
     <script type="text/javascript" src="${relative_path}build/app.min.js"></script>
     ${log}
+    <div class="copy content">Powered by Juno</div>
```
Comment (artem-zinnatullin) [inline html-report/index.html]: Maybe "Made by Juno with ❤️"?

File: html-report/postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    }),
+    require('cssnano')({
+      preset: ['default', {
+        discardComments: {
+          removeAll: true,
+        },
+      }]
+    })
+  ]
+}
```
Comment (artem-zinnatullin) [inline html-report/postcss.config.js]: Please enable `Ensure line feed at the end of file on Save` in WebStorm so it'll automatically add empty line at the end of file :)
screen shot 2017-06-07 at 00 43 14

File: html-report/src/components/SuitesList.js (hunk 1 above)
Comment (artem-zinnatullin) [inline html-report/src/components/SuitesList.js]: Ok

File: html-report/styles/app.css [hunk 4]
```diff
@@ -1333,5 +1324,9 @@ select.empty {
...
   color: #C200FF;
 }
-
-/*# sourceMappingURL=main.ae0a4831.css.map*/
+.copy {
+  color: #879bab;
+  font-size: 12px;
+  padding-bottom: 10px;
+  text-align: right;
+}
```
Comment (artem-zinnatullin) [inline html-report/styles/app.css]: Empty line pls

File: html-report/index.html (hunk 2 above)
Comment (artem-zinnatullin) [inline html-report/index.html]: Or "Generated with ❤️ by Juno"

Like GitHub does when you embed a gist somewhere 

screen shot 2017-06-07 at 00 52 55

Comment (artem-zinnatullin) [inline html-report/index.html]: And as we wanted in #36, we can add date here: 

"Generated with ❤️ by Juno at 00:56 UTC 7 June 2017"
`Generated with ❤️ by Juno ${date}`?

Comment (navstyachka) [inline html-report/index.html]: Sweet ^^ Will do

File: html-report/postcss.config.js (hunk 3 above)
Comment (navstyachka) [inline html-report/postcss.config.js]: 💛

**Pull request merged**